app/src/main/java/com/example/appempty/ViewModel/UserViewModel.kt

```diff
@@ -38,7 +38,7 @@ class UserViewModel : ViewModel() {
                 if (!response.isSuccessful) throw Throwable("error1")
 
                 val result: UserProfile = response.body()?.results?.firstOrNull() ?: UserProfile()
-                usuario.value = result
+                usuario.value = MutableList(100) { result }
```

Comment:
This should happen in the UserList, that is why you used result[0] and did not use the listaViewModel, because you are using only one ViewModel for both fragments, both logics should be on a separate ViewModels and attached to both Fragments.